pkgs/development/compilers/chicken/5/chicken.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchurl, makeWrapper, darwin, bootstrap-chicken ? null }:
+{ lib, stdenv, fetchurl, makeWrapper, targetPlatform, hostPlatform, darwin, bootstrap-chicken ? null }:
```

Comment:
I would prefer if we wouldn't allow override to null here.

Reply:
the code that calls this is 
```
chicken = callPackage ./chicken.nix {
  bootstrap-chicken = self.chicken.override { bootstrap-chicken = null; };
};
```
but removing the null `boostrap-chicken ? null` didn't  cause a rebuild so i guess its okay